subsys/mpsl/clock_ctrl/mpsl_clock_ctrl.c

```diff
@@ -326,6 +351,59 @@ static bool m_hfclk_is_running(void)
 
 	return true;
 }
+
+#if defined(CONFIG_MPSL_EXT_CLK_CTRL_NVM_CLOCK_REQUEST)
+
+static int32_t m_nvm_clock_request(void)
+{
+	int err;
+
+	/* In the MRAM DTS node there is no:
```

Comment:
I would slightly rephrase it:
```suggestion
	/* The MRAM DTS node does not provide:
```